apps/judicial-system/backend/src/app/modules/user/dto/updateUser.dto.ts

```diff
@@ -1,47 +1,53 @@
-import { IsBoolean, IsOptional, IsString } from 'class-validator'
+import {
+  IsBoolean,
+  IsEnum,
+  IsOptional,
+  IsString,
+  IsUUID,
+} from 'class-validator'
 
 import { ApiPropertyOptional } from '@nestjs/swagger'
 
-import type { UserRole } from '@island.is/judicial-system/types'
+import { UserRole } from '@island.is/judicial-system/types'
 
 export class UpdateUserDto {
   @IsOptional()
   @IsString()
-  @ApiPropertyOptional()
+  @ApiPropertyOptional({ type: String })
```

Comment:
I'm a bit confused, why is this an optional property, but still has a `!`?

Reply:
Good catch, all the fields should be optional.